src/main/java/com/jcabi/w3c/DefaultCssValidator.java

```diff
@@ -91,8 +100,10 @@ private ValidationResponse processed(final String css) throws IOException {
             this.uri,
             this.entity("file", DefaultCssValidator.filter(css), "text/css")
         );
+        final Response response = req.fetch();
```

Comment:
@jrdalpra let's get rid this line and one line below
it possible if you changed 
```
private void assertThatDoesNotHasBadHttpStatusAt(final Response response)
```
to 
```
private Response assertThatDoesNotHasBadHttpStatusAt(final Response response)
{
....
return response;
}
```